build.gradle

```diff
@@ -174,6 +174,12 @@ dependencies {
     integrationTestImplementation("org.testcontainers:junit-jupiter:${project.testcontainersVersion}")
 }
 
+jar {
+    doFirst {
+        new File("$projectDir/src/main/resources/sidecar.version").text = version
+    }
+}
+
```

Comment:
Does it create a version file with the content? 
What happens if the file already exist? 
Does it make sense to checkin the version file?

Reply:
It does, silently overwriting the file if it exists.
I don't think we should check the file in — having it change automatically may be distracting/confusing, and changing it manually will easy to forget.

Reply:
Added a comment explaining the current behavior.